flicks/views/movieGenres.py

```diff
@@ -0,0 +1,51 @@
+import json
+from django.http import HttpResponse, HttpResponseRedirect
+from django.views.decorators.csrf import csrf_exempt
+from django.utils.decorators import method_decorator
+from django.views import View
+import requests
+import os
+import random
+from rest_framework import status
+from ..models import Movie, Genre, User, UserFlick
+from ..serializers import  MovieGenres
+from dotenv import load_dotenv
+from django.db.models.functions import Lower
+import re
+
+
+@method_decorator(csrf_exempt, name="dispatch")
+class GetGenres(View):
+    def get(self, request):
+        load_dotenv()
+        API_KEY = os.getenv("MOVIES_DB_API_KEY")
+        genres_list = list()
+        response = dict()
+        statusCode = status.HTTP_200_OK
+        try:
+            print('GET MOVIE GENRES')
+            url = "https://api.themoviedb.org/3/genre/movie/list?language=en"
+
+            headers = {
+            "accept": "application/json",
+            "Authorization": "Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiIyNWE3MjkzMjg3MzA4MDY0OWU4M2ZmNzRhMDBhMjZkNiIsInN1YiI6IjY1NmZiODlmOTQ2MzE4MDEwMGM2YmE4ZiIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.37AzMiibDDo_bX1TMRmOeBJ-JwDHQFk1b8U9vEocq-Y"
+            }
```

Comment:
why is this token baked in?

Reply:
Cause API_KEY wouldnt work for some reason although the documentation said it would.

Reply:
I'm gonna fix this stuff when I get the Env deploy done